installation/install.sh

```
@@ -1568,7 +1568,7 @@ is_install_general_libs_rh(){
 
     # Fix upgrade dependency conflicts
     if [[ "$KYLIN_V10_OS" =~ "$ZSTACK_RELEASE" ]]; then
```

Shellcheck (0.10.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)